src/ostree/ot-builtin-checksum.c

```diff
@@ -94,9 +94,9 @@ ostree_builtin_checksum (int argc, char **argv, GCancellable *cancellable, GErro
       return data.success;
     }
 
-  char csum[OSTREE_SHA256_STRING_LEN+1];
+  g_autofree char *csum = NULL;
```

Comment:
Let's say `checksum` now too for consistency?  (I'm not sure I ever documented the naming scheme but it's *mostly* used consistently now)